securedrop/tests/functional/source_navigation_steps.py

```diff
@@ -161,5 +161,4 @@ def _source_deletes_a_journalist_reply(self):
 
     def _source_logs_out(self):
         self.driver.find_element_by_id('logout').click()
-        notification = self.driver.find_element_by_css_selector('.important')
-        assert 'Thank you for exiting your session!' in notification.text
+        self.driver.find_element_by_css_selector('.important')
```

Comment:
I added an explicit ``assert`` to make clear this is what we are verifying, good point.